### ISSUE TITLE
dojson: bug in hepnames/fields/bd1xx.py

### ISSUE THREAD

**Issue by kaplun** [inspirehep/inspire-next#414]
From https://travis-ci.org/inspirehep/inspire-next/builds/85131955#L129:
```python
  File "/usr/local/lib/python2.7/dist-packages/dojson/utils.py", line 46, in wrapper
    out = f(*args, **kwargs)
  File "/home/travis/build/inspirehep/inspire-next/inspire/dojson/hepnames/fields/bd1xx.py", line 311, in field_categories
    'name': value.get('a', '').lower(),
AttributeError: 'list' object has no attribute 'lower'
```

@jalavik @jmartinm @ioannistsanaktsidis

**Comment by kaplun** [comment]
BTW I just checked and this is the last bug that need to be solved before being able to load all the 8000 demo records in #357

**Comment by ioannistsanaktsidis** [comment]
I can take care of it tomorrow first thing in the morning. I think I know
the problem! ;)

2015-10-13 17:24 GMT+02:00 Samuele Kaplun <notifications@github.com>:

**Comment by kaplun** [comment]
:kissing_heart:

**Comment by kaplun** [comment]
Closed in b28d81ea614fe1b5ae36ebc3c36bd66b89aef1e3
